pkg/api/conversion.go

```diff
@@ -248,3 +252,16 @@ func Convert_unversioned_LabelSelector_to_map(in *metav1.LabelSelector, out *map
 	}
 	return err
 }
+
+func Convert_Slice_string_To_Slice_int32(in *[]string, out *[]int32, s conversion.Scope) error {
```

Comment:
It would be really useful to start documenting each custom conversion that we have: https://github.com/kubernetes/kubernetes/issues/39865